templates/common/_base/units/ovs-configuration.service.yaml

```diff
@@ -3,8 +3,6 @@ enabled: {{if eq .NetworkType "OVNKubernetes" "OpenShiftSDN"}}true{{else}}false{
 contents: |
   [Unit]
   Description=Configures OVS with proper host networking configuration
-  # Removal of this file signals firstboot completion
-  ConditionPathExists=!/etc/ignition-machine-config-encapsulated.json
```

Comment:
I think we can keep the condition here, but instead just add `After=machine-config-daemon-firstboot.service`.

Reply:
OK

Reply:
Tried to implement.  Got the following errors:
```
Aug 15 13:55:53 test-infra-cluster-087a552f-master-2 systemd[1]: machine-config-daemon-firstboot.service: Found ordering cycle on machine-config-daemon-pull.service/start
Aug 15 13:55:53 test-infra-cluster-087a552f-master-2 systemd[1]: machine-config-daemon-firstboot.service: Found dependency on network-online.target/start
Aug 15 13:55:53 test-infra-cluster-087a552f-master-2 systemd[1]: machine-config-daemon-firstboot.service: Found dependency on ovs-configuration.service/start
Aug 15 13:55:53 test-infra-cluster-087a552f-master-2 systemd[1]: machine-config-daemon-firstboot.service: Found dependency on machine-config-daemon-firstboot.service/start
Aug 15 13:55:53 test-infra-cluster-087a552f-master-2 systemd[1]: machine-config-daemon-firstboot.service: Job machine-config-daemon-pull.service/start deleted to break ordering cycle starting with machine-config-daemon-firstboot.service/start
Aug 15 13:56:04 test-infra-cluster-087a552f-master-2 systemd[1]: Configures OVS with proper host networking configuration was skipped because of an unmet condition check (ConditionPathExists=!/etc/ignition-machine-config-encapsulated.json).
```
Any idea how to address these errors ?  What is the right ordering for the services ?

Reply:
Ugh.  Right.  Our systemd units are just a huge mess in this respect; I hit similar network ordering issues in https://github.com/openshift/machine-config-operator/pull/3788

I think to fix it properly we need to get anything that depends on pulling containers to `After=network-online.target` - we need to introduce a new `kubelet-network-online.target`  and make just kubelet pull that in.